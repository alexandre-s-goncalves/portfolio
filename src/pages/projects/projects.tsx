import React from 'react';
import * as S from './projects.styles';

export const Projects = () => {
  return (
    <S.Container data-testid="projects-page">
      <S.Title>Projects</S.Title>
    </S.Container>
  );
};
