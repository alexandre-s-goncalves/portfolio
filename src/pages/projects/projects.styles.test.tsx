import React from 'react';
import {render} from '@testing-library/react';
import '@testing-library/jest-dom';
import 'jest-styled-components';
import * as S from './projects.styles';

describe('projects.styles', () => {
  test('Container DEVE ser igual ao snapshot', () => {
    const {container} = render(<S.Container />);
    expect(container.firstChild).toMatchSnapshot();
  });

  test('Title DEVE ser igual ao snapshot', () => {
    const {container} = render(<S.Title />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
