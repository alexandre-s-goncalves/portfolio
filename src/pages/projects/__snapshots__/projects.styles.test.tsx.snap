// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`projects.styles Container should match the snapshot 1`] = `
.c0 {
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100vh;
}

<div
  class="c0"
/>
`;

exports[`projects.styles Title should match the snapshot 1`] = `
.c0 {
  font-size: 24px;
  font-family: Inter-Bold;
  color: #434645;
  margin-top: 0px;
  font-weight: normal;
  text-transform: none;
  cursor: text;
}

<h1
  class="c0 "
/>
`;
